sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestPositionalBinder.java

```diff
@@ -17,39 +17,33 @@
 
 import java.util.List;
 import java.util.Map;
-import java.util.UUID;
 
-import org.h2.jdbcx.JdbcDataSource;
-import org.jdbi.v3.DBI;
+import org.jdbi.v3.H2DatabaseRule;
 import org.jdbi.v3.Handle;
-import org.junit.After;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
 
 public class TestPositionalBinder {
 
+    @Rule
+    public H2DatabaseRule db = new H2DatabaseRule();
+
     private Handle handle;
     private SomethingDao somethingDao;
 
     @Before
     public void setUp() throws Exception {
-        JdbcDataSource ds = new JdbcDataSource();
-        ds.setURL("jdbc:h2:mem:" + UUID.randomUUID());
-        DBI dbi = new DBI(ds);
-        handle = dbi.open();
+        handle = db.getSharedHandle();
         somethingDao = SqlObjectBuilder.attach(handle, SomethingDao.class);
 
+        handle.execute("drop table something");
```

Comment:
the rule creates it automagically, and this test uses a conflicting table definition, so we drop and recreate it.  not the best, but should be fixed as a separate PR